src/cmd/root_test.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 func TestIsAcceptableGitVersion(t *testing.T) {
+	t.Parallel()
```

Comment:
Consistent parallelization of unit tests reduces unit test time from 3s to 2.3 sec on my machine, i.e. 24% reduction.